include/seqan3/range/views/repeat.hpp

```diff
@@ -50,6 +50,20 @@ class repeat_view : public std::ranges::view_interface<repeat_view<value_t>>
     //!\brief The sentinel type is set to std::ranges::default_sentinel_t.
     using sentinel_type = std::ranges::default_sentinel_t;
 
```

Comment:
In the moment I dont get why these associated types are needed in random_access_iterator. Is that meant in the future or a different commit?